source/api_documentation/call_api/index.rst

```diff
@@ -0,0 +1,20 @@
+Call API
+================
+
+The Call API is accessible using the API credentials generated on the platform.
+
+.. raw:: html
+
+The data returned depends on your account type. To see which fields you will see, select an account type below.
```

Comment:
I'm not sure about the relevance of account type in the description here. We are looking at either recordings or transcripts, right? So request type may be better here